core/types/src/error.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+use alloc::string::String;
+
+/// Errors seen when converting to, or from, rust for the SGX types
+#[derive(Debug)]
+pub enum Error {
+    /// When the provided bytes or slice are too small to support converting
+    /// to the type.
+    InvalidInputLength,
+
+    /// Enum out of range
+    /// Happens when a value that is not represented by the known C enum values.
+    UnknownEnumValue(String),
```

Comment:
I'm strongly tempted to just say "i64" here :-)

Reply:
done